tools/qa/trapdoor_pydocstyle.py

```diff
@@ -65,9 +65,10 @@ def get_stats(self, config):
         print 'USING              : pydocstyle', version
 
         # Call pydocstyle in the directories containing Python code. All files will be
-        # checked, including test files. Missing tests are ignored (D103) because they are
-        # already detected by PyLint in a better way.
-        command = ['pydocstyle', '--match=.*\\.py', '--add-ignore=D103'] + \
+        # checked, including test files. Missing docstrings are ignored because they are
+        # detected by PyLint in a better way.
+        command = ['pydocstyle', '--match=.*\\.py',
+                   '--add-ignore=D100,D101,D102,D103,D104,D105'] + \
```

Comment:
Is this because of the abstract methods? It's disabling a lot. Maybe it would be better to add excludes at the head of the abstract files?